src/cap.rs

```diff
@@ -23,6 +23,8 @@ use kvm_bindings::*;
 #[allow(missing_docs)]
 pub enum Cap {
     Irqchip = KVM_CAP_IRQCHIP,
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    SplitIrqchip = KVM_CAP_SPLIT_IRQCHIP,
```

Comment:
The capabilities are sorted by value. Please move this lower (should be between `KVM_CAP_S390_USER_SIGP` and `KVM_CAP_IMMEDIATE_EXIT`).

Reply:
Fixed